input/pages/codings.xml

```diff
@@ -10,10 +10,10 @@
 {% endcomment %} -->
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hl7.org/fhir ../../src-generated/schemas/fhir-single.xsd">
 
-  <p>
-    The following is an alphabetical table of external coding systems often used in the genomics space, with FHIR CodeSystem URIs and OIDs where available. See <a href="{{site.data.fhir.path}}terminologies-systems.html">the core specification guidance</a> for more details.
-  </p>
-
+    <p>Genomics covers a broad landscape involving many organizations outside of HL7 that have established codes and name systems for different use cases. The <a href="https://confluence.hl7.org/display/TA">HL7 Terminology Authority</a> governs the use of terminologies that are external to HL7 and they have begun to document these. As a part of the <a href="https://confluence.hl7.org/display/TSMG">Terminology Services Management Group</a>, the HTA is working with these organizations to establish canonical URIs that can be used as HL7 code and name systems. HL7 has established <a href="https://terminology.hl7.org">terminology.hl7.org (THO)</a> as the authoritative source for code system information, including <a href="https://terminology.hl7.org/external_terminologies.html">external terminologies</a> Many of these are are also listed in the <a href="{{site.data.fhir.path}}terminologies-systems.html">the core specification guidance</a>.</p>
+    <p>We have listed many, but not all, of genomics-related systems below. In this table, we provide links for more information -- to the THO or HTA entries when available, or to organization home page if not. We also list a URI that has been approved by the THO/HTA or by <a href="https://confluence.hl7.org/display/VOC/Vocabulary+Work+Group">Vocabulary Work Group</a>, and a coding system label defined in the <a href="http://www.hl7.org/implement/standards/product_brief.cfm?product_id=279">HL7 Version 2.5.1 Implementation Guide: Laboratory Results Interface</a> (LRI).</p>
+    <p>For those organizations that list the URI as "not assigned yet," check the <a href="https://terminology.hl7.org/external_terminologies.html">THO</a> or <a href="https://confluence.hl7.org/display/TA/External+Terminologies+-+Information">HTA sites</a> to see if it has been added since the publication of this IG. If not, we recommend following the <a href="https://confluence.hl7.org/display/TA/External+Code+System+Owner+Engagement+Process">External Code System Owner Engagement Processing</a>. As a last resort, we suggest using the organization site URL, change any "https" to "http", and remove trailing slashes.</p>
+    <p>Please contact the <a href="http://www.hl7.org/Special/committees/clingenomics/leadership.cfm">Clinical Genomics Workgroup</a> co-chairs if you have any questions about using these genomics code systems.</p>
```

Comment:
Should we use this link instead?
https://confluence.hl7.org/display/CGW

Reply:
I considered that, but the other site actually lists email addresses and a link to send email to all co-chairs. btw, I couldn't find where the vocab facilitator is listed. But I don't have a problem with linking to confluence.

Reply:
I would prefer linking to confluence, and if needed, we improve the confluence page.

Reply:
done